src/components/ResultsView.tsx

```diff
@@ -597,6 +597,13 @@ const ResultsView: ResultsViewComponent = ({
                               value,
                               blockUid: layoutNode.uid,
                             });
+                            if (s.options === "columns") {
+                              setInputSettings({
+                                key: "columns",
+                                blockUid: layoutNode.uid,
+                                values: [],
+                              });
+                            }
```

Comment:
Not *super* happy with hardcoding `"columns"` in here again ... if you have any suggestions, I'm all ears.

Reply:
This is for clearing out a layout setting when certain layout specific settings are updated.
specifically these two for Kanban
```
      { key: "key", label: "Group By", options: "columns" },
      { key: "display", label: "Display", options: "columns" },
```
RE: subtree
I wonder how ugly the backwards compatibility would be for this with multiple settings already

RE: `clearOnSwitch` 
I like this, I think this would work